sdcm/nemesis.py

```diff
@@ -2474,27 +2475,23 @@ def break_streaming_task_and_rebuild(self, task='decommission'):  # pylint: disa
         """
 
         def decommission_post_action():
-            ips = []
-            seed_nodes = [node for node in self.cluster.nodes if node.is_seed]
-            status = self.cluster.get_nodetool_status(verification_node=seed_nodes[0])
-            self.log.debug(status)
-            for dc_info in status.values():
-                try:
-                    ips.extend(dc_info.keys())
-                except Exception:  # pylint: disable=broad-except
-                    self.log.debug(dc_info)
-
-            decommission_done = list(self.target_node.follow_system_log(
-                patterns=['DECOMMISSIONING: done'], start_from_beginning=True))
+            target_is_seed = self.target_node.is_seed
+            try:
+                self.cluster.verify_decommission(self.target_node)
+            except NodeStayInClusterAfterDecommission:
+                return None
+            else:
```

Comment:
why do you log error in `else` block, "else" block is entered when there was no exception...

Reply:
Good catch, I will change it to `except:`, which will catch other kinds of unexpected exception.